test/framework/src/main/java/org/opensearch/test/OpenSearchSingleNodeTestCase.java

```diff
@@ -246,9 +247,14 @@ private Node newNode() {
             .putList(DISCOVERY_SEED_HOSTS_SETTING.getKey()) // empty list disables a port scan for other nodes
             .putList(INITIAL_CLUSTER_MANAGER_NODES_SETTING.getKey(), nodeName)
             .put(FeatureFlags.TELEMETRY_SETTING.getKey(), true)
-            .put(TelemetrySettings.TRACER_ENABLED_SETTING.getKey(), true)
-            .put(nodeSettings()) // allow test cases to provide their own settings or override these
-            .build();
+            .put(TelemetrySettings.TRACER_ENABLED_SETTING.getKey(), true);
+        if (FeatureFlags.isEnabled(FeatureFlags.CONCURRENT_SEGMENT_SEARCH)) {
```

Comment:
@sohami sorry, question here: with this test setup, we would never exercise the Apache Lucene's default flow, do you think we could randomize that:

```
if (randomBoolean() && FeatureFlags.isEnabled(FeatureFlags.CONCURRENT_SEGMENT_SEARCH)) {
}
```
?

Reply:
The difference between lucene default flow and custom slice computation is how slices are computed. The concurrent search part still remains same over those slices. For IT tests, we are more interested in concurrent search part and this custom mechanism will increase that probability.

Reply:
> For IT tests, we are more interested in concurrent search part and this custom mechanism will increase that probability.

I understand that but we won't be testing the flow that vast majority of the users will experience - that is more important than experimental feature - I am trying to find out the balance here

Reply:
> I understand that but we won't be testing the flow that vast majority of the users will experience

There are 2 parts to concurrent search: 1) computing slices (this is dependent on data layout and size) and 2) executing search on those concurrent slices. For (1) Default lucene mechanism will create multiple slices in real world datasets only because of its default value of segmentCount=5 and docCount=250k. Also the segment distribution among slices will affect mostly performance not functionality which is what we want to verify in IT tests. Hope that clarifies.

Reply:
Trust you here :) thanks @sohami